packages/@tinacms/cli/src/tinaLocalLevel/index.ts

```diff
@@ -0,0 +1,13 @@
+import { ManyLevelGuest } from 'many-level'
+import { pipeline } from 'readable-stream'
+import { connect } from 'net'
+
+export class TinaLocalLevel extends ManyLevelGuest {
+  constructor() {
+    super()
+    const socket = connect(9000)
+    pipeline(socket, this.createRpcStream(), socket, () => {
+      // Disconnected
+    })
```

Comment:
this probably should be in a `connect()` function instead

Reply:
Where would connect get called?

Reply:
Would we call it when we are indexing? and then it could be called when the user wants to fetch data?

Reply:
Same place we call the existing connect function, so you'd just instantiate the object and call connect on it:
```
const localLevelStore = new TinaLevelClient()
localLevelStore.connect()
```

Reply:
I'm not sure - the database would need to accept any level implementation - so I don't think we could put it in `connect()`. Maybe overriding open() and putting it in there makes sense (assuming open() gets called automatically somewhere before its actually needed)

Reply:
Hmm when I put it in open I get this error.
```
Error: Only one rpc stream can be active
```

Reply:
Added a new function. I called it `openConnection` as to not have any naming conflicts